cmd/env/updatetracecfg.go

```diff
@@ -46,7 +46,7 @@ func init() {
 		"", "Trace endpoint, used only with JAEGER")
 	UpdateTraceConfigCmd.Flags().StringVarP(&sampler, "sampler", "s",
 		"PROBABILITY", "Sampler can be set to PROBABILITY or OFF")
-	UpdateTraceConfigCmd.Flags().StringVarP(&sampleRate, "rate", "r",
+	UpdateTraceConfigCmd.Flags().StringVarP(&sampleRate, "rate", "",
```

Comment:
why was this removed? Is it because of the global `r` flag? If yes, then removing the existing `r` for `rate` is breaking change right?